Хмелев Дмитрий

NeSmogPridymatNick opened the pull request.

NeSmogPridymatNick committed:
стили

NeSmogPridymatNick committed:
верстка 5

NeSmogPridymatNick committed:
исправлены ошибки codestyle

NeSmogPridymatNick committed:
исправлены ошибки codestyle

NeSmogPridymatNick committed:
картинки

xiiivii commented:
@NeSmogPridymatNick посмотрел
Ни одно из обязательных заданий не выполнено.
Список выглядит не так как на макетах, длинные имена не обрезаются (см. лекцию по текстам), по наведению ничего не происходит

Надо бы доделать

P.S. медиазапросы – хорошо

NeSmogPridymatNick committed:
Добавлены недостающие задания

Вопрос.
Почему не раотает такая конструкция 
div img{ 
float: left; 
} 
input[value = 't']:checked ~ img 
{ 
float: none; 
}

NeSmogPridymatNick committed:
Добавлены недостающие задания

xiiivii commented on index.css:
```diff
@@ -1 +1,84 @@
+    margin: 10px;
+    padding: 0;
+    border: 0;
+    display: inline-table;
```
нужно было использовать `inline-block`, переделай под них

xiiivii commented on index.html:
```diff
@@ -1,11 +1,206 @@
+    <link href="index.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+    <input type="radio" name="cat" value="t" checked>Таблица<br>
```
напрашиваются `label`ы

xiiivii commented:
@NeSmogPridymatNick остаилсь принципиальные проблемы, в том числе упавшие тесты, а именно:
список выглядит не так как на макетах, длинные имена не обрезаются

xiiivii commented:
@NeSmogPridymatNick даю время до понедельника сдать эту работу, в понедельник она будет закрыта непринятой
